staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go

```diff
@@ -36,6 +37,15 @@ var ForeverTestTimeout = time.Second * 30
 // NeverStop may be passed to Until to make it never stop.
 var NeverStop <-chan struct{} = make(chan struct{})
 
+// StartUntil starts f in a new goroutine and calls done once f has finished.
+func StartUntil(stopCh <-chan struct{}, wg *sync.WaitGroup, f func(stopCh <-chan struct{})) {
```

Comment:
I appreciate you taking the feedback into account. I [left a comment](https://github.com/kubernetes/kubernetes/pull/46094/commits/94093074b1f3896531c25b4f69f52e395f6837ac#r120114634) on 94093074b1f3896531c25b4f69f52e395f6837ac.